db/import_listener.go

```diff
@@ -37,6 +37,7 @@ func (il *importListener) StartImportFeed(bucket base.Bucket, dbStats *base.DbSt
 		ID:         base.DCPImportFeedID,
 		Backfill:   sgbucket.FeedResume,
 		Terminator: il.terminator,
+		DoneChan:   make(chan struct{}),
```

Comment:
It doesn't look like you're doing anything to block/wait on this channel for import feed processing - does this channel need to be initialized here?

Reply:
As discussed, this is currently needed for the CE case, where we're using the simple StartDCPFeed, to avoid panic closing a nil channel.  It might be safer to have goroutines only close DoneChan when it's non-nil.

Reply:
Added changes to have goroutines only close DoneChan when it's non-nil.